src/lib/Behat/Page/MyDraftsPage.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\AdminUi\Behat\Page;
+
+use Behat\Mink\Session;
+use Ibexa\AdminUi\Behat\Component\Dialog;
+use Ibexa\AdminUi\Behat\Component\Table\TableBuilder;
+use Ibexa\AdminUi\Behat\Component\Table\TableInterface;
+use Ibexa\Behat\Browser\Locator\VisibleCSSLocator;
+use Ibexa\Behat\Browser\Page\Page;
+use Ibexa\Behat\Browser\Routing\Router;
+
+final class MyDraftsPage extends Page
+{
+    private TableInterface $table;
+
+    private Dialog $dialog;
+
+    public function __construct(Session $session, Router $router, TableBuilder $tableBuilder, Dialog $dialog)
+    {
+        parent::__construct($session, $router);
+        $this->table = $tableBuilder->newTable()->build();
+        $this->dialog = $dialog;
+    }
+
+    public function verifyIsLoaded(): void
+    {
+    }
+
+    public function deleteDraft(string $draftName): void
+    {
+        $this->table->getTableRow(['Name' => $draftName])->select();
+
+        $this->getHTMLPage()->find($this->getLocator('deleteButton'))->click();
+        $this->dialog->confirm();
+    }
+
+    public function doSeeDraft(string $draftName): bool
```

Comment:
```suggestion
    public function isDraftOnTheList(string $draftName): bool
```

We can use https://github.com/ibexa/admin-ui/blob/main/src/lib/Behat/Component/Table/TableInterface.php#L15 here, meaning that we should be able to do this:

```php
public function isDraftOnTheList(string $draftName): bool
{
    return $this->table->hasElement(['Name' => $draftName]);
}
```